src/main/resources/overriding.properties

```diff
@@ -1,2 +1,3 @@
-# Put here properties that will be used for local debugging and local test runs.
-# The properties defined in this file override any properties set in dedicated folders
+bdd.story-loader.batch-1.resource-location=story/web
```

Comment:
`overriding.properties` should not be in the repository or your working branch at all. It shouldn't be pushed